farmhouse/theme.json

```diff
@@ -0,0 +1,599 @@
+{
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"palette": [
+				{
+					"color": "#fbfcec",
+					"name": "Base",
+					"slug": "base"
+				},
+				{
+					"color": "#216609",
+					"name": "Contrast",
+					"slug": "contrast"
+				},
+				{
+					"color": "#216609",
+					"name": "Primary",
+					"slug": "primary"
+				},
+				{
+					"color": "#216609",
+					"name": "Secondary",
+					"slug": "secondary"
+				},
+				{
+					"color": "#e1e5c4",
+					"name": "Tertiary",
+					"slug": "tertiary"
+				}
+			]
+		},
+		"layout": {
+			"contentSize": "620px",
+			"wideSize": "1000px"
+		},
+		"spacing": {
+			"units": [
+				"%",
+				"px",
+				"em",
+				"rem",
+				"vh",
+				"vw"
+			]
+		},
+		"typography": {
+			"fluid": true,
+			"fontFamilies": [
+				{
+					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+					"name": "System Font",
+					"slug": "system-font"
+				},
+				{
+					"fontFace": [
+						{
+							"fontFamily": "\"DM Sans\"",
+							"fontStyle": "normal",
+							"fontWeight": "300",
+							"src": "https://farmhouse.mystagingwebsite.com/wp-content/uploads/fonts/rP2tp2ywxg089UriI5-g4vlH9VoD8CmcqZG40F9JadbnoEwA_JxhS2f3ZGMZpg.woff2"
```

Comment:
Looks like these font asset paths need to be updated to be relative to the theme assets folder.

Reply:
Done!